packages/rum/src/types.ts

```diff
@@ -12,6 +12,7 @@ export interface SegmentMeta extends SegmentContext {
   has_full_snapshot: boolean
   records_count: number
   creation_reason: CreationReason
+  sequence_number: number
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** `‏sequence_number` seems a bit blurry to me, it could be more precise with a naming indicating that it is related to the view, something like `view_index` or `position_in_view`, wdyt?

Reply:
This is something we agreed on with @acorretti . What do you think about those alternate namings Alfonso? Any preference?

Reply:
Well they're not really _views_, but I'm okay with something more specific like `segment_index`

Reply:
After discussion with @acorretti , we agreed on `index_in_view`. I'll make the change